homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -157,11 +159,6 @@ async def async_added_to_hass(self):
         if self._state and self._off_delay is not None:
             self._state = False
 
-    @property
-    def force_update(self) -> bool:
-        """We should force updates. Repeated states have meaning."""
-        return True
-
     @property
     def is_on(self):
```

Comment:
Could you migrate `is_on` to the attribute, by updating `self._attr_is_on` instead of `self._state`?

Reply:
Yup, done. Was just a larger change.

Reply:
Oups - I was looking forward to a nice little review 😜